### PR TITLE
Merge 1.7.x

### PR DESCRIPTION
**Pull request by rhattersley**

### CODE REVIEW

File: docs/iris/src/whatsnew/1.7.rst [hunk 1]
```diff
@@ -194,6 +194,13 @@ Bugs fixed
 * Add missing NetCDF attribute `varname` when constructing a new :class:`iris.coords.AuxCoord`.
 * The datatype of time arrays converted with :func:`iris.util.unify_time_units` is now preserved.
 
+Bugs fixed in v1.7.3
```
Comment (esc24) [inline docs/iris/src/whatsnew/1.7.rst]: Interesting that the date for the release is `???`

Comment (rhattersley) [inline docs/iris/src/whatsnew/1.7.rst]: Presumably because as a preparation for the 1.7.3 release.

Comment (esc24) [inline docs/iris/src/whatsnew/1.7.rst]: But 1.7.3 was released on Dec 16th (http://www.scitools.org.uk/iris/docs/latest/whatsnew/1.7.html)

Comment (rhattersley) [inline docs/iris/src/whatsnew/1.7.rst]: Ah. Ooops!

**Pull request merged**